app/tree/gui/src/ts/twxplore/gui/geo/components/TreeCollapse/TreeCollapse.tsx

```diff
@@ -0,0 +1,112 @@
+import './TreeCollapse.scss';
+import * as React from "react";
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import {ListGroup, ListGroupItem} from "reactstrap";
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+//import * as PropTypes from "prop-types";
+
+
+
+type Feature_List = {
+  createdAt: string,
+  dbh: number,
+  stump: number,
+  curbLoc: string,
+  status: string,
+  health: string | null,
+  species: string | null,
+  steward: string | null,
+  guards: string | null,
+  sidewalk: string | null,
+  userType: string,
+  problems: string[],
+  address: string,
+  city: string,
+  community: number,
+  cncldist: number,
+  stateAssembly: number,
+  stateSenate: number,
+  boroughCount: number,
+  latitude: number,
+  longitude: number,
+  x_sp: number,
+  y_sp: number,
+  bin: number | null,
+  bbl: any | null,
+  uri: string
+}
+
+type TreeCardProps = {
```

Comment:
Don't really need this type since it's only declared in the function component signature. React.FunctionComponent<{ features: Feature_List, callSetMode: Function }>